cowprotocol/accounting/rewards/mainnet/partner_fees_query_3602560.sql

```diff
@@ -0,0 +1,77 @@
+-- This query computes the partner fees associated with each partner in a given time interval
+--
+-- Parameters:
+--  {{start_time}} - the timestamp for which the accounting should start (inclusively)
+--  {{end_time}} - the timestamp for which the accounting should end (exclusively)
+--  {{result}} - two views of the result, one aggregated and one on a per tx basis
+
+with
+per_settlement_prelim as (
```

Comment:
this is probably not per settlement but per trade
```suggestion
per_trade as (
```